label_studio/tests/data_manager/api_tasks.tavern.yml

```diff
@@ -102,238 +102,307 @@ stages:
         "total": 4,
```

Comment:
XXX: very odd behavior  in my environment. 
if this test is executed standalone
```pytest -k tasks-all-fields-postgre -vv tests/data_manager/api_tasks.tavern.yml```
it passes

if it's executed as part of the test suite it fails! it creates one less prediction
```
    "total_annotations": 4,
    "total_predictions": 2,
    "total": 3,
```